### PR TITLE
Judy

### PR DESCRIPTION
**Pull request by judyzhang25**
Basic models for Drone and Orders with some testing.

### CODE REVIEW

File: Dronuts/common/models/order.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    'user_id': {
+        'type': 'string',
+        'required': true
+    },
+    'items': {
+        'type': ['string'],
+        'required': true
+    },
+    'destination_lat': {
+        'type': 'number',
```
Comment (derekyyyan) [inline Dronuts/common/models/order.js]: Does `number` mean float/double? If it's some kind of integer then it wouldn't make sense for latitude and longitude.

File: Dronuts/test/drones.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+require('jest-plugins')(['loopback-jest']);
+var loopback = require('loopback');
+
+describe('basic drone test', () => {
+    test('drone exists and has properties', ( ) => {
+        const drone1 = loopback.createModel({
+            name: 'Drone',
+            properties: {
+                drone_id: 'string'
```
Comment (derekyyyan) [inline Dronuts/test/drones.test.js:9]: I'm not sure if this code is defining a `Drone` model or creating an instance? If it's the latter, then I suggest changing the value of `drone_id` to some string that doesn't look like it's defining the datatype (e.g. `test_string`).

File: Dronuts/common/models/order.js (hunk 1 above)
Comment (judyzhang25) [reply Dronuts/common/models/order.js]: I believe number is the most generic loopback type. There are no specific float or double types.

**Comment by derekyyyan** [comment]
lgtm

**Pull request merged**